### PR TITLE
Ntrnl 374 update GitHub api sdk methods

### PR DESCRIPTION
**Pull request by Mouhajer-CO**
### JIRA link

Resolves [NTRNL-374](https://technologyprogramme.atlassian.net/browse/NTRNL-374)

### Description

- Update module version and fix vulnerability
- Update responseHandler and add getGitHubInfo, postIssue and getCollaboratorsPerRepo calls

### Work checklist

- [x] Tests added where applicable
- [x] No vulnerability added

### CODE REVIEW

File: src/api-sdk/github/github.ts [hunk 1]
```diff
@@ -1,37 +1,75 @@
...
 import { HttpRequest } from '../../http-request';
 
 export class Github {
-    constructor(private readonly request: HttpRequest) {
-        /**/
+
+    constructor(private readonly request: HttpRequest) { /**/ }
+
+    // Get info data. Could be repos/members/collaborator/teams ...
+    public async getGitHubInfo(url: string): Promise<ApiResponse<any[]> | ApiErrorResponse> {
```
Comment (harley-harris) [inline src/api-sdk/github/github.ts:27]: Could implement a union type for `GitHubInfo` which includes all the info interfaces? Something like:

`export type GitHubInfo = GitHubRepos | GitHubTeams | GitHubMembers | GitHubCollaboratorsPerRepo`

**Comment by harley-harris** [comment]
`README.md` also needs updating to reflect refactors.

File: src/api-sdk/github/github.ts (hunk 1 above)
Comment (Mouhajer-CO) [reply src/api-sdk/github/github.ts:27]: The returned object does not have a well-defined data type. I have not found an official `npm` `@type` module for it, and it is not directly mapped by us, which prevents the use of our internal interface

Comment (harley-harris) [reply src/api-sdk/github/github.ts:27]: Fair enough, leave as `any`

**Review by harley-harris — APPROVED** [review]
Great work Mo again 🚀

**Pull request merged**